core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -515,7 +519,7 @@ public Future<?> submit(Runnable task) {
   public ScheduledThreadPoolExecutor
       createGeneralScheduledExecutorService(AccumuloConfiguration conf) {
     return (ScheduledThreadPoolExecutor) createExecutorService(conf,
-        Property.GENERAL_SIMPLETIMER_THREADPOOL_SIZE, true);
+        Property.GENERAL_THREADPOOL_SIZE, true);
```

Comment:
I'm not sure this is sufficient. The old property is deprecated, not removed. This means that users may have the old property, and not the new one, in their configuration. I think you need to check for both and maybe use the one that has a larger value.

Reply:
May want to look at what other properties are doing. I thought we had a replacement mechanism that other code had used.

Reply:
Implemented in 4a6bd0eac6bd5de996976caf8870b4b87e7daea0.